outputs/redis.go

```diff
@@ -0,0 +1,63 @@
+package outputs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/falcosecurity/falcosidekick/types"
+	"github.com/redis/go-redis/v9"
+)
+
+func NewRedisClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics,
+	statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+
+	ctx := context.Background()
+
+	rClient := redis.NewClient(&redis.Options{
+		Addr:     config.Redis.Address + ":" + config.Redis.Port,
+		Password: config.Redis.Password,
+		DB:       config.Redis.Database,
+	})
+
+	// Ping the Redis server to check if it's running
+	pong, err := rClient.Ping(ctx).Result()
+	if err != nil {
+		log.Println(err)
+	}
+	fmt.Println(pong, "Redis Server Running")
+
+	return &Client{
+		OutputType:      "Redis",
+		Config:          config,
+		RedisClient:     rClient,
+		Stats:           stats,
+		PromStats:       promStats,
+		StatsdClient:    statsdClient,
+		DogstatsdClient: dogstatsdClient,
+	}, nil
+}
+
+func (c *Client) RedisPost(falcopayload types.FalcoPayload) {
+	c.Stats.Redis.Add(Total, 1)
+
+	ctx := context.Background()
+
+	redisPayload, _ := json.Marshal(falcopayload)
```

Comment:
I'm not a Redis expert, but I understand we store the json as a simple string. Can we think about allowing other storage formats, like direct a `JSON` with https://redis.io/docs/stack/json/ or `hash(map)` https://redis.com/redis-best-practices/data-storage-patterns/object-hash-storage/.
The choice of the format could be set by the user too.

Reply:
@Issif yes, We can. Thanks for the review, I will add the option for this well. I can set this in a config and set default as a list (which is currently there.)